src/main/java/org/fundacionjala/coding/kevincristhian/BinaryArrayToNumber.java

```diff
@@ -0,0 +1,31 @@
+package org.fundacionjala.coding.kevincristhian;
+
+import java.util.List;
+
+/**
+ * Ones and Zeros 7kyu.
+ */
+public final class BinaryArrayToNumber {
+
+    /**
+     * constructor.
+     */
+    private BinaryArrayToNumber() {
+    }
+
+    /**
+     * method to test.
+     * @param binary value
+     * @return int
+     */
+    public static int convertBinaryArrayToInt(final List<Integer> binary) {
+      int dim = binary.size();
+      int i = 0;
+      int value = 0;
+      while (dim > i) {
+          value += binary.get(i) * (int) (Math.pow(2, dim - i - 1));
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Cast one of the operands of this subtraction operation to a "double". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2184)